amti/clis/review.py

```diff
@@ -42,6 +47,7 @@ def review_batch(batch_dir, live, approve_all):
     actions.review.review_batch(
         client=client,
         batch_dir=batch_dir,
-        approve_all=approve_all)
+        approve_all=approve_all,
+        mark_file=mark_file)
```

Comment:
Changing the `mark_file` to `mark_file_path` (my comment at amti/actions/review.py#L130) will also require changing it here.